src/readinglistbar.cpp

```diff
@@ -11,10 +11,13 @@
 const QString documentsDir = QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation);
 
 ReadingListBar::ReadingListBar(QWidget *parent) :
-    QWidget(parent),
+    QFrame(parent),
     ui(new Ui::readinglistbar)
 {
     ui->setupUi(this);
+    ui->titleLabel->setFont(QFont("Selawik", 18, QFont::Weight::Medium));
```

Comment:
Why is the font set in C++ code? Can't it be controlled via CSS?

Reply:
Yes it can be set via CSS but TOC also had font set in cpp code in tableofcontentbar.cpp... so to maintain code consistency i added this line here in readinglistbar.cpp.

Reply:
If possible I am ok with doing this in CSS. Its just my personal preference to control font in Qt, as I remember some widgets doesn't support CSS fonts.

Reply:
Hello.. I just checked it's working via CSS, so should i do it in CSS??

Reply:
Yes, please